internal/driver/glfw/driver_desktop.go

```diff
@@ -46,6 +46,12 @@ func (d *gLDriver) SetSystemTrayMenu(m *fyne.Menu) {
 				d.SetSystemTrayIcon(theme.BrokenImageIcon())
 			}
 
+			title := fyne.CurrentApp().Metadata().Name
+			if title == "" {
+				title = fyne.CurrentApp().UniqueID()
```

Comment:
The app lookup uses atomics to get the variable. Might be better to avoid doing so twice. 
```suggestion
			app := fyne.CurrentApp()
			title := app.Metadata().Name
			if title == "" {
				title = app.UniqueID()
```

Reply:
Good catch, thanks!